delete.js

```diff
@@ -1,25 +1,24 @@
 {
-	let newChattyObj = Object.create(null);
+	let removeMessage = Object.create(null);
 	let deleteButton = document.getElementsByClassName("delete-btn");
 
 	// One IIFE should accept a message element id and then remove the correct element from the DOM.
 	// This IIFE should also remove the corresponding message from the private array that was created
 	// in the previous IIFE.
-	newChattyObj.removeMessage = function(id) {
+	removeMessage.removeFromDOM = function(id) {
 		let currentElement = document.getElementById(id);
 		currentElement.parentNode.remove();
 	}
 
-	newChattyObj.removeFromArray = function(msgArray, id) {
+	removeMessage.removeFromArray = function(msgArray, id) {
```

Comment:
i was thinking it was less vague (what does new chatty obj mean? vs this iife's functionality is about removing the messages from the DOM and array. i was also thinking it would make things consistent (that's more the naming convention that was used in the other js files. but it's not important we can totally leave them the way it was.